datafusion/expr-common/src/accumulator.rs

```diff
@@ -64,9 +64,6 @@ pub trait Accumulator: Send + Sync + Debug {
     /// For example, the `SUM` accumulator maintains a running sum,
     /// and `evaluate` will produce that running sum as its output.
     ///
-    /// After this call, the accumulator's internal state should be
```

Comment:
I think it would be better update the docs to explain what the state can be after calling the function rather than simply removing the docs. 

For example, is it the case that 
* "calling evaluate twice should result in an error"?
* "calling evaluate twice will result in potentially non deterministic behavior"?
* Something else?

Reply:
Agree concise documentation is quite important. Especially when DF becoming more feature rich and code is evolving

Reply:
Added "Not call twice" comment. Feel free to help me polish comment wording.